upbound_uxp/README.md

```diff
@@ -0,0 +1,163 @@
+GCP are out of the box supported cloud providers.
+
+This Datadog integration facilitates the monitoring
+of Universal Crossplane and provider Kubernetes pods.
+The value of observing the metrics is to be aware of
+the health of this self managed infrastructure provisioning
+and resource lifecycle management ecosystem to rest assured
+that the infrastructure is properly managed, or that actions
+may be needed to correct its configuration. The metrics
+inform about performance and resource consumption. This
+helps validating the Univeral Crossplane management cluster
+sizing and enables optimization.
+
+The checks monitor [Upbound UXP](https://docs.upbound.io/uxp/)
+related metrics.
+
+It looks for UXP and provider pods by default
+in the upbound-system Kubernetes
+namespace. The Upbound and provider
+pods emit metrics at port 8080/metrics in
+a Prometheus compatible format.
+
+## Setup
+
+### Prerequisites
+
+The Datadog agent requires permissions to discover UXP and provider pods.
+Create a service account, a cluster role, and a cluster role binding
+to grant those permissions.
+
+Note that the datadog agent in the configuration
+below resides in an arbitrary `monitoring` namespace.
+Your agent may reside in a namespace of your choice.
+Configure the service account and cluster role accordingly.
+
+```
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: datadog-upbound
+  namespace: monitoring
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: apiserver-cluster-role
+  namespace: monitoring
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs: ["get", "list" ]
+  - nonResourceURLs: ["/metrics"]
+    verbs: ["get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: apiserver-cluster-role-binding
+subjects:
+- namespace: monitoring
+  kind: ServiceAccount
+  name: datadog-upbound
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: apiserver-cluster-role
+```
+
+### Installation
+
+To install the Upbound UXP check on your host:
+
+
+1. Install the [developer toolkit]
+(https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit)
+ on any machine.
+
+2. Run `ddev release build upbound_uxp` to build the package.
+
+3. [Download the Datadog Agent]["Datadog to supply the URL for the agent location"].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/upbound_uxp/dist/<ARTIFACT_NAME>.whl` or copy the build artifact
+ to an agent container, that may be running in Kubernetes with `kubectl cp
+ path/to/upbound_uxp/dist/<ARTIFACT_NAME>.whl -n <MONITORING_NAMESPACE>
+ <DATADOG_CONTAINER_NAME>:<PATH_TO_DIST_IN_CONTAINER>` and then install
+ it with `kubectl -n <MONITORING_NAMESPACE> exec -it <DATADOG_CONTAINER_NAME>
+ -- agent integration install -w -r
+ <PATH_TO_DIST_IN_CONTAINER>/<ARTIFACT_NAME>.whl`.
```

Comment:
Can you please break this out into sub-bullets for each command?

Reply:
Structured into sub sections.